src/AEG/CutNode.ts

```diff
@@ -115,32 +115,23 @@ export class CutNode {
     }
 
     /**
-     * Method that checks whether the given node can be inserted into this cut
-     * at a given point without overlapping any bounding boxes.
-     * @param incomingNode The node to be inserted.
-     * @param insertionPoint The point at which the node must be inserted
-     * @returns True, if the node can be inserted. Else, false
+     * Checks how many layers deep the new node can be placed.
```

Comment:
Typically the word "checks" signals a function returning a bool
```suggestion
     * Computes how many layers deep the new node can be placed.
```